### ISSUE TITLE
Two unit-tests fail "wrongly".

### ISSUE THREAD

**Issue by KerimG** [patrickjuchli/basic-ftp#86]
**Describe the bug**
Two unit tests fail, even though they shouldn't: 

```
  83 passing (202ms)
  2 failing

  1) Download directory listing
       handles destination stream error:

      AssertionError [ERR_ASSERTION]: 'Error: EISDIR: illegal operation on a directory, open \'C:\\Users\\KG\\code\\basic-ftp\\test\'' == 'Error: EISDIR: illegal operation on a directory, open \'test\''
      + expected - actual

      -Error: EISDIR: illegal operation on a directory, open 'C:\Users\KG\code\basic-ftp\test'
      +Error: EISDIR: illegal operation on a directory, open 'test'

      at client.download.catch.err (test\downloadSpec.js:176:20)

  2) Upload
       handles error events from the source stream:

      AssertionError [ERR_ASSERTION]: 'Error: ENOENT: no such file or directory, open \'C:\\Users\\KG\\code\\basic-ftp\\nothing.txt\'' == 'Error: ENOENT: no such file or directory, open \'nothing.txt\''
      + expected - actual

      -Error: ENOENT: no such file or directory, open 'C:\Users\KG\code\basic-ftp\nothing.txt'
      +Error: ENOENT: no such file or directory, open 'nothing.txt'

      at client.upload.catch.err (test\uploadSpec.js:114:20)
```

It seems the actual error contains the absolute path to the error'd file, while the assertions expect just the filename. Obviously one can't assert an absolute path in the unit tests but perhaps check whether the error in case of 2) contains: the string

```
Error: ENOENT: no such file or directory, open
```
as well as the filename?

**Which version of Node.js are you using?**
Windows 10
Node 10.15.0

**Comment by patrickjuchli** [comment]
Interesting, maybe this is specific to Windows? They don't break in the CI of this repo.

We can just leave away the check for the actual filename. The unit test is about testing that an error is thrown, not so much what the exact message is. I'll have a look.

**Comment by patrickjuchli** [comment]
Will be released in 3.8.1
